### PR TITLE
Login Epilogue: adjust table width for iPad

### PR DESCRIPTION
**Pull request by ScoutHarris**
Ref #13413 

This updates the Login Epilogue's site list table width based on the size class and device orientation.
- For the Regular x Regular size class:
  - In portrait, the table is 66% of the view width.
  - In landscape, the table is 50% of the view width.
- For all other size classes, the table is 100% of the view width.

In addition, the Done button is now always the same width as the table.

To test:

---
- Login on an iPad.
- Verify the table and button width are sized properly in portrait and landscape.

| ipad_portrait | ipad_landscape |
|--------|-------|

---
- Login on an iPhone, or multitask on an iPad and change the view size.
- Verify the table and button width are sized properly.

iphone

---
PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by ScoutHarris** [comment]
> there isn't a uniform left alignment

🤦 

Thanks @mattmiklic ! 

That's what I get for focusing too much on the site list. I forgot to update the User Info and Connect Site cells accordingly. It shall be done shortly!

**Comment by ScoutHarris** [comment]
Hey @mattmiklic . Sorry about that. The alignment has been fixed. I've update the screenshots in the description. Thanks!

File: WordPress/Classes/ViewRelated/NUX/LoginEpilogueViewController.swift [hunk 1]
```diff
@@ -132,6 +150,29 @@ private extension LoginEpilogueViewController {
...
+                tableViewLeadingConstraint.constant = defaultTableViewMargin
+                tableViewTrailingConstraint.constant = defaultTableViewMargin
+                return
+        }
+
+        let marginMultiplier = UIDevice.current.orientation.isLandscape ?
+            TableViewMarginMultipliers.ipadLandscape :
+            TableViewMarginMultipliers.ipadPortrait
+
+        let margin = view.frame.width * marginMultiplier
```
Comment (mindgraffiti) [inline WordPress/Classes/ViewRelated/NUX/LoginEpilogueViewController.swift:166]: The view hasn't been updated when this function is called, so the rotation was reported but the view's frame will have the old orientation's numbers.

Comment (mindgraffiti) [inline WordPress/Classes/ViewRelated/NUX/LoginEpilogueViewController.swift:166]: If you delete these lines and add this:

```
let isLandscape = UIDevice.current.orientation.isLandscape
        let marginMultiplier = isLandscape ?
        TableViewMarginMultipliers.ipadLandscape :
        TableViewMarginMultipliers.ipadPortrait

        let width = isLandscape ? view.frame.height : view.frame.width

        let margin = width * marginMultiplier
```

it should work.

Comment (ScoutHarris) [reply WordPress/Classes/ViewRelated/NUX/LoginEpilogueViewController.swift]: Thanks for the help @mindgraffiti !

Your suggestion helped a lot, but for me getting the `width` still wasn't always correct (I suspect for the exact same reason my approach wasn't always correct - as you stated the view hasn't been updated when this function is called).

> let width = isLandscape ? view.frame.height : view.frame.width

So I determined the width "manually", i.e.: 
```
let screenSizeMax = max(view.frame.width, view.frame.height)
let screenSizeMin = min(view.frame.width, view.frame.height)
let viewWidth = isLandscape ? screenSizeMax : screenSizeMin
```

This worked for me consistently.

**Comment by ScoutHarris** [comment]
Hey @mindgraffiti . I've made a couple changes. The table width should now be calculated correctly. Ready for another review when you can. Thanks!

**Comment by ScoutHarris** [comment]
Thanks @mindgraffiti !

**Pull request merged**